### ISSUE TITLE
OktaApiError type doesn't match response from Okta.Client

### ISSUE THREAD

**Issue by wallactp** [okta/okta-sdk-nodejs#374]
### Describe the bug?

The `OktaApiError` [type](https://github.com/okta/okta-sdk-nodejs/blob/master/src/types/api-error.d.ts) structure doesn't match the response I'm receiving from the Okta.Client. The `errorCauses` property is currently set as a string, but I am receiving an array of objects.

Example:
If I try to create an oidc application with an existing clientId, I expect an error because clientIds can't be duplicated. I do receive that error
```
{
  status: 400,
  errorCode: 'E0000001',
  errorSummary: 'Api validation failed: client_id',
  errorCauses: [
    {
      errorSummary: 'client_id: An object with this field already exists in the current organization'
    }
  ],
  errorLink: 'E0000001',
  errorId: 'random id',
  url: 'https://my.okta.url/api/v1/apps',
  headers: [Object]
}
```

The error structure for `errorCauses` doesn't match the `OktaApiError` type
```
export declare class OktaApiError extends Error {
  constructor(url: string, status: number, responseBody: Record<string, unknown>, headers: Headers);
  name: string;
  status: number;
  errorCode: string | number;
  errorSummary: string;
  errorCauses: string;
  errorLink: string;
  errorId: string;
  url: string;
  headers: Headers;
  stack: string;
  message: string;
}
```

### What is expected to happen?

I would like to be able to use the `OktaApiError` type to type any errors received from the okta client.

### What is the actual behavior?

I cannot use `OktaApiError` to type errors because `errorCauses` has the wrong type.

### Reproduction Steps?

Make a request the will fail API validation. In my case, I tried to create an Application that doesn't use a unique client_id.

### SDK Versions

"@okta/okta-sdk-nodejs": "6.6.0"

### Execution Environment

nodejs

**Comment by denysoblohin-okta** [comment]
Thanks for submitting this issue.
Internal ref: OKTA-572970

**Comment by denysoblohin-okta** [comment]
Fixed in 7.0.1
